sdk/communication/azure-communication-callingserver/azure/communication/callingserver/aio/_call_connection_async.py

```diff
@@ -126,6 +128,53 @@ async def remove_participant(
             **kwargs
         )
 
+    @distributed_trace_async()
+    async def cancel_participant_media_operation(
+            self,
+            participant_id,  # type: str
+            media_operation_id,  # type: str
+            **kwargs  # type: Any
+        )-> None:
+
+        if not participant_id:
+            raise ValueError("participant_id can not be None")
+
+        if not media_operation_id:
+            raise ValueError("media_operation_id can not be None")
+
+        cancel_media_operation_request = CancelMediaOperationRequestConverter.convert(
+            media_operation_id=media_operation_id
+            )
+
+        return await self._call_connection_client.cancel_participant_media_operation(
+            call_connection_id=self.call_connection_id,
+            participant_id=participant_id,
+            cancel_media_operation_request=cancel_media_operation_request,
+            **kwargs
+        )
+
+    @distributed_trace_async()
+    async def transfer_call(
+            self,
```

Comment:
Async method are python3 supported only, we should follow python3 convention

 async def transfer_call(
            self,
            target_participant: CommunicationIdentifier,
            user_to_user_information: Optional[str],
            **kwargs: Any
        ) -> None: